resource/group/group.go

```diff
@@ -42,7 +42,7 @@ type Group struct {
 
 // SystemUtils provides system utilities for group
 type SystemUtils interface {
-	AddGroup(string, string) error
+	AddGroup(string, map[string]string) error
```

Comment:
could you add a comment to explain why this needs two arguments? Or just name the arguments?